tests/unit_tests/run_e2e_tests.py.in

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env python3
+import sys
+import pytest
+
+if __name__ == "__main__":
+    sys.path.append("${CMAKE_BINARY_DIR}/diffkemp/simpll")
+    sys.exit(pytest.main([
+        "-x",
+        "${CMAKE_BINARY_DIR}/../",
```

Comment:
Maybe we could add there the `tests` directory and we would not have to ignore the kernels?
```suggestion
        "${CMAKE_BINARY_DIR}/../tests",
```